src/components/VideoControls/VideoControls.js

```diff
@@ -33,18 +32,26 @@ const VideoControls = (props) => {
             video.pause();
         }
     };
+    const stopVideo = () => (video.currentTime = video.duration);
 
     useEffect(() => {
         video.addEventListener('loadedmetadata', () => {
+            const timeLeftOff = localStorage.getItem(videoName);
```

Comment:
can we prefix the localStorage key with something unique? So something like:
```suggestion
            const timeLeftOff = localStorage.getItem(`vid--name--${videoName}`);
```

this way, we'll have more control over the keys stored regardless of the video name